wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -125,6 +125,9 @@ impl PhysicalDeviceFeatures {
                 //.shader_int64(requested_features.contains(wgt::Features::SHADER_INT64))
```

Comment:
@kvark could we get a list of these unimplemented features somewhere?

Reply:
https://github.com/gfx-rs/wgpu/wiki/Extensions